pkgs/top-level/all-packages.nix

```diff
@@ -3179,7 +3179,10 @@ with pkgs;
 
   lilo = callPackage ../tools/misc/lilo { };
 
-  logseq = callPackage ../applications/misc/logseq { };
+  logseq = callPackage ../applications/misc/logseq {
+    # graphs will not sync without matching upstream's major electron version
+    electron = electron_27;
```

Comment:
We should probably move that back. It is also more cross friendly to keep this in the file.